server/src/data/models.rs

```diff
@@ -439,7 +439,7 @@ impl From<ChunkMetadataWithFileData> for SlimChunkMetadata {
     }
 }
 
-#[derive(Debug, Serialize, Deserialize, Clone, Queryable)]
+#[derive(Debug, Serialize, Deserialize, Clone, Queryable, ToSchema)]
```

Comment:
Line below, but mayybe change to `ScoreSlimChunks` or something? 

I changed from it OnlyId